pkg/bu_server/trade_document/file_based_ebl_allow_action.go

```diff
@@ -76,7 +76,14 @@ func IsFileEBLPrintable(bl *bill_of_lading.BillOfLadingPack, bu string, withDeta
 }
 
 func IsFileEBLTransferable(bl *bill_of_lading.BillOfLadingPack, bu string, withDetail bool) error {
-	return model.ErrEBLActionNotAllowed
+	if bu != bl.CurrentOwner {
+		if withDetail {
+			return fmt.Errorf("not the current owner. %w", model.ErrEBLActionNotAllowed)
+		}
+		return model.ErrEBLActionNotAllowed
+	}
+
```

Comment:
Not every role can do "Transfer".  We need to check which role the business unit is.